aten/src/ATen/native/cpu/LerpKernel.cpp

```diff
@@ -0,0 +1,62 @@
+#include <ATen/ATen.h>
+
+#include <ATen/Dispatch.h>
+#include <ATen/native/Lerp.h>
+#include <ATen/native/TensorIterator.h>
+#include <ATen/native/cpu/Loops.h>
+
+namespace at {
+namespace native {
+namespace {
+
+static void lerp_kernel_scalar(
+    Tensor& ret,
+    const Tensor& self,
+    const Tensor& end,
+    Scalar weight) {
+  auto builder = at::TensorIterator::Builder();
+  builder.add_output(ret);
+  builder.add_input(self);
+  builder.add_input(end);
+  auto iter = builder.build();
+  AT_DISPATCH_FLOATING_TYPES(ret.scalar_type(), "lerp_kernel_scalar", [&] {
+    scalar_t weight_val = weight.to<scalar_t>();
+    at::native::cpu_kernel(
+        *iter,
+        [=](scalar_t self_val, scalar_t end_val) {
```

Comment:
I always prefer explicitly listing the variables you're capturing for something like this, because it'll be called a lot and it could be a costly mistake.